### PR TITLE
Document texture3d, add some colorspace references

### PR DESCRIPTION
**Pull request by luisbarrancos**
To address https://github.com/appleseedhq/appleseed-maya/issues/144

### CODE REVIEW

File: docs/source/shaders/texture/as_texture3d.rst [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+*T Wrap*
+    The wrapping mode along the *t* coordinate, it can be one of
+
+        * Default
+        * Black
+        * Periodic
+        * Clamp
+        * Mirror
+
+*T Wrap*
```
Comment (dictoon) [inline docs/source/shaders/texture/as_texture3d.rst]: Duplicate block?

File: docs/source/shaders/texture/as_texture3d.rst [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+-------
+
+*Output Color*
+    The color resulting from ghe *Features Mode* choice.
+
+-----
+
+.. rubric:: Footnotes
+
+.. [#] These files are exported as a result of simulations done in other applications. In the case of Maya, one could for instance convert Maya's fluids to a field3d file via the `Field3D Maya Plugin <https://github.com/magic-box/Field3DMayaPlugin>`_. Other simulation applications also allow exporter volume data as *\*.f3d*.
```
Comment (dictoon) [inline docs/source/shaders/texture/as_texture3d.rst]: *Field3D*

File: docs/source/shaders/texture/as_texture3d.rst (hunk 1 above)
Comment (luisbarrancos) [reply docs/source/shaders/texture/as_texture3d.rst]: fixed below with https://github.com/appleseedhq/appleseed-maya/pull/145/commits/173be6dc0185d5430ae34344d175886b8fe92f53

File: docs/source/shaders/texture/as_texture3d.rst [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        * World Space
+        * Camera Space
+
+-----
+
+Outputs
+-------
+
+*Output Color*
+    The color resulting from ghe *Features Mode* choice.
```
Comment (dictoon) [inline docs/source/shaders/texture/as_texture3d.rst]: Typo: *the*

**Pull request merged**